diff -Nur --exclude='.*' tiramisu/CMakeLists.txt tiramisu_patched/CMakeLists.txt
--- tiramisu/CMakeLists.txt	2021-10-14 10:40:01.000000000 +0400
+++ tiramisu_patched/CMakeLists.txt	2021-10-14 16:30:54.000000000 +0400
@@ -180,7 +180,7 @@
     link_directories(${MPI_BUILD_DIR}/lib)
     include_directories(${MPI_BUILD_DIR}/include)
 else ()
-    set(CMAKE_CXX_COMPILER "g++")
+    set(CMAKE_CXX_COMPILER "${CMAKE_CXX_COMPILER}")
 endif()
 
 
diff -Nur --exclude='.*' tiramisu/benchmarks/compile_and_run_benchmarks.sh tiramisu_patched/benchmarks/compile_and_run_benchmarks.sh
--- tiramisu/benchmarks/compile_and_run_benchmarks.sh	2021-10-14 10:40:02.000000000 +0400
+++ tiramisu_patched/benchmarks/compile_and_run_benchmarks.sh	2021-10-14 10:37:40.000000000 +0400
@@ -1,7 +1,5 @@
 #!/bin/bash
 
-CXX=g++
-
 #set -x
 
 if [ $# -eq 0 ]; then
diff -Nur --exclude='.*' tiramisu/utils/scripts/install_submodules.sh tiramisu_patched/utils/scripts/install_submodules.sh
--- tiramisu/utils/scripts/install_submodules.sh	2021-10-14 10:40:03.000000000 +0400
+++ tiramisu_patched/utils/scripts/install_submodules.sh	2021-10-14 10:36:14.000000000 +0400
@@ -40,6 +40,7 @@
 echo_and_run_cmd "cd ${PROJECT_SRC_DIR}"
 echo_and_run_cmd "git submodule update --init --remote --recursive"
 
+sed -i '' -e '32s/^//p; 32s/^.*/#include <sys\/syslimits.h>/' ${PROJECT_SRC_DIR}/3rdParty/Halide/src/Util.cpp
 
 # Get ISL installed
 echo "#### Installing isl ####"
